config/201-clusterrole.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ceph-source-controller
+  labels:
+    ceph.eventing.knative.dev/release: devel
+rules:
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs: &everything
+  - get
+  - list
+  - watch
+  - create
+  - update
+  - patch
+  - delete
+
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterroles
+  verbs:
+  - list
+
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs: *everything
+
+- apiGroups:
+  - sources.knative.dev
+  resources:
+  - cephsources
+  verbs: *everything
+
+- apiGroups:
+  - sources.knative.dev
+  resources:
+  - cephsources/status
+  - cephsources/finalizers
+  verbs:
+  - get
+  - update
+  - patch
+
+- apiGroups:
+    - sources.knative.dev
+  resources:
+    - sinkbindings
+  verbs: *everything
+- apiGroups:
+    - sources.knative.dev
+  resources:
+    - sinkbindings/status
+  verbs: *everything
+
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+
```

Comment:
Our reconciler needs `coordination.k8s.io/lease` cluster-role for leader-election.
Like I did in the https://github.com/knative-sandbox/sample-source/pull/169/files .